backend/api/Areas/Activities/Controllers/ActivityController.cs

```diff
@@ -76,6 +86,29 @@ public IActionResult GetActivityById(long activityId)
             return new JsonResult(_mapper.Map<ActivityInstanceModel>(activity));
         }
 
+        /// <summary>
+        /// Retrieves the activity with the specified id.
+        /// </summary>
+        /// <param name="fileType">The type of file the fileId belongs to.</param>
+        /// <param name="fileId">the id of the file.</param>
+        /// <returns></returns>
+        [HttpGet("{fileType}/file/{fileId:long}")]
```

Comment:
This looks like it should retrieve the activities for the specified file id. Is that the intent?